extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+                "type": "ARRAY",
+                "componentType": "UINT8",
+                "componentCount": 3
+              },
+              "priority": {
+                "type": "UINT32"
+              }
+            }
+          }
+        }
+      },
+      "groups": {
+        "buildings": {
+          "class": "layer",
+          "properties": {
+            "color": [128, 128, 128],
+            "priority": 0
+          }
+        },
+        "trees": {
+          "class": "layer",
+          "properties": {
+            "color": [10, 240, 30],
+            "priority": 1
+          }
+        }
+      }
+    }
+  },
+  "root": {
+    "refine": "ADD",
+    "geometricError": 32768.0,
+    "boundingVolume": {
+      "region": [-1.707, 0.543, -1.706, 0.544, -10.3, 253.113]
+    },
+    "children": [
+      {
+        "geometricError": 0.0,
+        "boundingVolume": {
+          "region": [-1.707, 0.543, -1.706, 0.544, -10.3, 253.113]
+        },
+        "content": {
+          "uri": "buildings.b3dm",
+          "extensions": {
+            "3DTILES_metadata": {
+              "group": "buildings"
+            }
+          }
+        }
+      },
+      {
+        "geometricError": 0.0,
+        "boundingVolume": {
+          "region": [-1.707, 0.543, -1.706, 0.544, -10.3, 253.113]
+        },
+        "content": {
+          "uri": "trees.i3dm",
+          "extensions": {
+            "3DTILES_metadata": {
+              "group": "trees"
+            }
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+### Feature Metadata
+
+A feature is an entity that has both geometry and metadata, such as a 3D building in a city. Features are the finest level of granularity within the tileset.
+
+Metadata may be assigned to features using the glTF extension [`EXT_feature_metadata`](https://github.com/CesiumGS/glTF/tree/master/extensions/2.0/Vendor/EXT_feature_metadata/1.0.0).
+
+Feature metadata classes may be included in the `3DTILES_metadata` schema. This lets an application know what classes exist before requesting content.
+
+### Statistics
+
+Statistics provide aggregate information about entities in a tileset. Statistics are provided on a per-class basis.
+
+* `count` is the number of entities that conform to the class
+* `properties` contains statistics about property values
+
+Properties have the following built-in statistics:
+
+Name|Description|Type
+--|--|--
+`min`|The minimum property value|Numeric types or fixed-length arrays of numeric types
+`max`|The maximum property value|...
+`mean`|The arithmetic mean of the property values|...
+`median`|The median of the property values|...
+`standardDeviation`|The standard deviation of the property values|...
+`variance`|The variance of the property values|...
+`sum`|The sum of the property values|...
+`occurrences`|Number of enum occurrences|Enums or fixed-length arrays of enums
+
+Tileset authors may define their own additional semantics, like `mode` in the example below.
+
+```jsonc
```

Comment:
This is an extremely long example to show one line of JSON:

`"mode": 5.0`

In general, don't force the reader to sort through unimportant detail to get to the essence of your message.